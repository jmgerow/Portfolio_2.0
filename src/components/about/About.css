#profile-pic {
    margin-right: 20px;
    margin-top: 20px;
    border-radius: 10px;
}