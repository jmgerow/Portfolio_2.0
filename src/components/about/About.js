import React, { Component } from 'react';

import profile from '../../images/profile.jpg';

import './About.css';

class About extends Component {

    render() {
        return (
            <div className="container">
                <div className="row blue-grey-text text-darken-1" id="title">
                    <h3>About Me</h3>
                </div>

                <div className="row">
                    <div className="col sm 12" id="bio">
                        <img className="left" src={profile} id="profile-pic" alt="Justin Gerow"></img>

                        <p>Justin Gerow, based in Chicago, is a Full Stack Web Developer who graduated from the
                            Northwestern University Coding Boot Camp Full Stack Development program.</p>


                        <p>Justin has spent the past 18 years working primarily in the transportation, warehousing, and
                            logistics fields. He started his career in 2000 as a package operations supervisor for a
                            small-parcel company located in Flint, Michigan. Justin spent eight years in this role and during
                            this time also attended the University of Michigan-Flint, where he obtained his Bachelor of
                            Business Administration degree, with a concentration in operations management. In May 2008, he
                            relocated to Irving, Texas, to join a large online retail organization as an area manager in a
                            local fulfillment center. In September 2009, Justin joined a transportation company as a driver
                            business leader based in Gary, Indiana, and in 2014 moved into his current role with the same
                            company, as a sales operations manager based in Chicago. In 2016, he received his Certified Supply
                            Chain Professional (CSCP) certification from APICS, the association for supply chain management.</p>


                        <p>Originally from Michigan, Justin enjoys participating in outdoor activities such as running and
                            hiking in his free time, as well as cooking and exploring Chicago with his girlfriend, Lindsey, and
                            10-year-old Husky, Link. He also enjoys following auto racing — specifically, rally racing and the
                            World Endurance Championship — and playing the occasional video game on those rainy days. Coding and
                            development is a passion of Justin’s, and he looks forward to applying his knowledge
                            in an exciting, evolving industry. </p>



                    </div>
                </div>
            </div>
        );
    };
};

export default About;