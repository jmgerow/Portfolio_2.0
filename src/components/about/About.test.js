import React from 'react';
import { shallow } from 'enzyme';
import About from './About';

describe('About', () => {
    const about = shallow(<About />)
  
    it('renders without crashing', () => {
      expect(about).toMatchSnapshot();
    });
 
  });