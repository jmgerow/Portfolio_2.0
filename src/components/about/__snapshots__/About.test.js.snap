// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders without crashing 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <About />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="row blue-grey-text text-darken-1"
          id="title"
        >
          <h3>
            About Me
          </h3>
        </div>,
        <div
          className="row"
        >
          <div
            className="col sm 12"
            id="bio"
          >
            <img
              alt="Justin Gerow"
              className="left"
              id="profile-pic"
              src="profile.jpg"
            />
            <p>
              Justin Gerow, based in Chicago, is a Full Stack Web Developer who graduated from the Northwestern University Coding Boot Camp Full Stack Development program.
            </p>
            <p>
              Justin has spent the past 18 years working primarily in the transportation, warehousing, and logistics fields. He started his career in 2000 as a package operations supervisor for a small-parcel company located in Flint, Michigan. Justin spent eight years in this role and during this time also attended the University of Michigan-Flint, where he obtained his Bachelor of Business Administration degree, with a concentration in operations management. In May 2008, he relocated to Irving, Texas, to join a large online retail organization as an area manager in a local fulfillment center. In September 2009, Justin joined a transportation company as a driver business leader based in Gary, Indiana, and in 2014 moved into his current role with the same company, as a sales operations manager based in Chicago. In 2016, he received his Certified Supply Chain Professional (CSCP) certification from APICS, the association for supply chain management.
            </p>
            <p>
              Originally from Michigan, Justin enjoys participating in outdoor activities such as running and hiking in his free time, as well as cooking and exploring Chicago with his girlfriend, Lindsey, and 10-year-old Husky, Link. He also enjoys following auto racing — specifically, rally racing and the World Endurance Championship — and playing the occasional video game on those rainy days. Coding and development is a passion of Justin’s, and he looks forward to applying his knowledge in an exciting, evolving industry. 
            </p>
          </div>
        </div>,
      ],
      "className": "container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <h3>
            About Me
          </h3>,
          "className": "row blue-grey-text text-darken-1",
          "id": "title",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "About Me",
          },
          "ref": null,
          "rendered": "About Me",
          "type": "h3",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="col sm 12"
            id="bio"
          >
            <img
              alt="Justin Gerow"
              className="left"
              id="profile-pic"
              src="profile.jpg"
            />
            <p>
              Justin Gerow, based in Chicago, is a Full Stack Web Developer who graduated from the Northwestern University Coding Boot Camp Full Stack Development program.
            </p>
            <p>
              Justin has spent the past 18 years working primarily in the transportation, warehousing, and logistics fields. He started his career in 2000 as a package operations supervisor for a small-parcel company located in Flint, Michigan. Justin spent eight years in this role and during this time also attended the University of Michigan-Flint, where he obtained his Bachelor of Business Administration degree, with a concentration in operations management. In May 2008, he relocated to Irving, Texas, to join a large online retail organization as an area manager in a local fulfillment center. In September 2009, Justin joined a transportation company as a driver business leader based in Gary, Indiana, and in 2014 moved into his current role with the same company, as a sales operations manager based in Chicago. In 2016, he received his Certified Supply Chain Professional (CSCP) certification from APICS, the association for supply chain management.
            </p>
            <p>
              Originally from Michigan, Justin enjoys participating in outdoor activities such as running and hiking in his free time, as well as cooking and exploring Chicago with his girlfriend, Lindsey, and 10-year-old Husky, Link. He also enjoys following auto racing — specifically, rally racing and the World Endurance Championship — and playing the occasional video game on those rainy days. Coding and development is a passion of Justin’s, and he looks forward to applying his knowledge in an exciting, evolving industry. 
            </p>
          </div>,
          "className": "row",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <img
                alt="Justin Gerow"
                className="left"
                id="profile-pic"
                src="profile.jpg"
              />,
              <p>
                Justin Gerow, based in Chicago, is a Full Stack Web Developer who graduated from the Northwestern University Coding Boot Camp Full Stack Development program.
              </p>,
              <p>
                Justin has spent the past 18 years working primarily in the transportation, warehousing, and logistics fields. He started his career in 2000 as a package operations supervisor for a small-parcel company located in Flint, Michigan. Justin spent eight years in this role and during this time also attended the University of Michigan-Flint, where he obtained his Bachelor of Business Administration degree, with a concentration in operations management. In May 2008, he relocated to Irving, Texas, to join a large online retail organization as an area manager in a local fulfillment center. In September 2009, Justin joined a transportation company as a driver business leader based in Gary, Indiana, and in 2014 moved into his current role with the same company, as a sales operations manager based in Chicago. In 2016, he received his Certified Supply Chain Professional (CSCP) certification from APICS, the association for supply chain management.
              </p>,
              <p>
                Originally from Michigan, Justin enjoys participating in outdoor activities such as running and hiking in his free time, as well as cooking and exploring Chicago with his girlfriend, Lindsey, and 10-year-old Husky, Link. He also enjoys following auto racing — specifically, rally racing and the World Endurance Championship — and playing the occasional video game on those rainy days. Coding and development is a passion of Justin’s, and he looks forward to applying his knowledge in an exciting, evolving industry. 
              </p>,
            ],
            "className": "col sm 12",
            "id": "bio",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "Justin Gerow",
                "className": "left",
                "id": "profile-pic",
                "src": "profile.jpg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Justin Gerow, based in Chicago, is a Full Stack Web Developer who graduated from the Northwestern University Coding Boot Camp Full Stack Development program.",
              },
              "ref": null,
              "rendered": "Justin Gerow, based in Chicago, is a Full Stack Web Developer who graduated from the Northwestern University Coding Boot Camp Full Stack Development program.",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Justin has spent the past 18 years working primarily in the transportation, warehousing, and logistics fields. He started his career in 2000 as a package operations supervisor for a small-parcel company located in Flint, Michigan. Justin spent eight years in this role and during this time also attended the University of Michigan-Flint, where he obtained his Bachelor of Business Administration degree, with a concentration in operations management. In May 2008, he relocated to Irving, Texas, to join a large online retail organization as an area manager in a local fulfillment center. In September 2009, Justin joined a transportation company as a driver business leader based in Gary, Indiana, and in 2014 moved into his current role with the same company, as a sales operations manager based in Chicago. In 2016, he received his Certified Supply Chain Professional (CSCP) certification from APICS, the association for supply chain management.",
              },
              "ref": null,
              "rendered": "Justin has spent the past 18 years working primarily in the transportation, warehousing, and logistics fields. He started his career in 2000 as a package operations supervisor for a small-parcel company located in Flint, Michigan. Justin spent eight years in this role and during this time also attended the University of Michigan-Flint, where he obtained his Bachelor of Business Administration degree, with a concentration in operations management. In May 2008, he relocated to Irving, Texas, to join a large online retail organization as an area manager in a local fulfillment center. In September 2009, Justin joined a transportation company as a driver business leader based in Gary, Indiana, and in 2014 moved into his current role with the same company, as a sales operations manager based in Chicago. In 2016, he received his Certified Supply Chain Professional (CSCP) certification from APICS, the association for supply chain management.",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Originally from Michigan, Justin enjoys participating in outdoor activities such as running and hiking in his free time, as well as cooking and exploring Chicago with his girlfriend, Lindsey, and 10-year-old Husky, Link. He also enjoys following auto racing — specifically, rally racing and the World Endurance Championship — and playing the occasional video game on those rainy days. Coding and development is a passion of Justin’s, and he looks forward to applying his knowledge in an exciting, evolving industry. ",
              },
              "ref": null,
              "rendered": "Originally from Michigan, Justin enjoys participating in outdoor activities such as running and hiking in his free time, as well as cooking and exploring Chicago with his girlfriend, Lindsey, and 10-year-old Husky, Link. He also enjoys following auto racing — specifically, rally racing and the World Endurance Championship — and playing the occasional video game on those rainy days. Coding and development is a passion of Justin’s, and he looks forward to applying his knowledge in an exciting, evolving industry. ",
              "type": "p",
            },
          ],
          "type": "div",
        },
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="row blue-grey-text text-darken-1"
            id="title"
          >
            <h3>
              About Me
            </h3>
          </div>,
          <div
            className="row"
          >
            <div
              className="col sm 12"
              id="bio"
            >
              <img
                alt="Justin Gerow"
                className="left"
                id="profile-pic"
                src="profile.jpg"
              />
              <p>
                Justin Gerow, based in Chicago, is a Full Stack Web Developer who graduated from the Northwestern University Coding Boot Camp Full Stack Development program.
              </p>
              <p>
                Justin has spent the past 18 years working primarily in the transportation, warehousing, and logistics fields. He started his career in 2000 as a package operations supervisor for a small-parcel company located in Flint, Michigan. Justin spent eight years in this role and during this time also attended the University of Michigan-Flint, where he obtained his Bachelor of Business Administration degree, with a concentration in operations management. In May 2008, he relocated to Irving, Texas, to join a large online retail organization as an area manager in a local fulfillment center. In September 2009, Justin joined a transportation company as a driver business leader based in Gary, Indiana, and in 2014 moved into his current role with the same company, as a sales operations manager based in Chicago. In 2016, he received his Certified Supply Chain Professional (CSCP) certification from APICS, the association for supply chain management.
              </p>
              <p>
                Originally from Michigan, Justin enjoys participating in outdoor activities such as running and hiking in his free time, as well as cooking and exploring Chicago with his girlfriend, Lindsey, and 10-year-old Husky, Link. He also enjoys following auto racing — specifically, rally racing and the World Endurance Championship — and playing the occasional video game on those rainy days. Coding and development is a passion of Justin’s, and he looks forward to applying his knowledge in an exciting, evolving industry. 
              </p>
            </div>
          </div>,
        ],
        "className": "container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h3>
              About Me
            </h3>,
            "className": "row blue-grey-text text-darken-1",
            "id": "title",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "About Me",
            },
            "ref": null,
            "rendered": "About Me",
            "type": "h3",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <div
              className="col sm 12"
              id="bio"
            >
              <img
                alt="Justin Gerow"
                className="left"
                id="profile-pic"
                src="profile.jpg"
              />
              <p>
                Justin Gerow, based in Chicago, is a Full Stack Web Developer who graduated from the Northwestern University Coding Boot Camp Full Stack Development program.
              </p>
              <p>
                Justin has spent the past 18 years working primarily in the transportation, warehousing, and logistics fields. He started his career in 2000 as a package operations supervisor for a small-parcel company located in Flint, Michigan. Justin spent eight years in this role and during this time also attended the University of Michigan-Flint, where he obtained his Bachelor of Business Administration degree, with a concentration in operations management. In May 2008, he relocated to Irving, Texas, to join a large online retail organization as an area manager in a local fulfillment center. In September 2009, Justin joined a transportation company as a driver business leader based in Gary, Indiana, and in 2014 moved into his current role with the same company, as a sales operations manager based in Chicago. In 2016, he received his Certified Supply Chain Professional (CSCP) certification from APICS, the association for supply chain management.
              </p>
              <p>
                Originally from Michigan, Justin enjoys participating in outdoor activities such as running and hiking in his free time, as well as cooking and exploring Chicago with his girlfriend, Lindsey, and 10-year-old Husky, Link. He also enjoys following auto racing — specifically, rally racing and the World Endurance Championship — and playing the occasional video game on those rainy days. Coding and development is a passion of Justin’s, and he looks forward to applying his knowledge in an exciting, evolving industry. 
              </p>
            </div>,
            "className": "row",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <img
                  alt="Justin Gerow"
                  className="left"
                  id="profile-pic"
                  src="profile.jpg"
                />,
                <p>
                  Justin Gerow, based in Chicago, is a Full Stack Web Developer who graduated from the Northwestern University Coding Boot Camp Full Stack Development program.
                </p>,
                <p>
                  Justin has spent the past 18 years working primarily in the transportation, warehousing, and logistics fields. He started his career in 2000 as a package operations supervisor for a small-parcel company located in Flint, Michigan. Justin spent eight years in this role and during this time also attended the University of Michigan-Flint, where he obtained his Bachelor of Business Administration degree, with a concentration in operations management. In May 2008, he relocated to Irving, Texas, to join a large online retail organization as an area manager in a local fulfillment center. In September 2009, Justin joined a transportation company as a driver business leader based in Gary, Indiana, and in 2014 moved into his current role with the same company, as a sales operations manager based in Chicago. In 2016, he received his Certified Supply Chain Professional (CSCP) certification from APICS, the association for supply chain management.
                </p>,
                <p>
                  Originally from Michigan, Justin enjoys participating in outdoor activities such as running and hiking in his free time, as well as cooking and exploring Chicago with his girlfriend, Lindsey, and 10-year-old Husky, Link. He also enjoys following auto racing — specifically, rally racing and the World Endurance Championship — and playing the occasional video game on those rainy days. Coding and development is a passion of Justin’s, and he looks forward to applying his knowledge in an exciting, evolving industry. 
                </p>,
              ],
              "className": "col sm 12",
              "id": "bio",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "Justin Gerow",
                  "className": "left",
                  "id": "profile-pic",
                  "src": "profile.jpg",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Justin Gerow, based in Chicago, is a Full Stack Web Developer who graduated from the Northwestern University Coding Boot Camp Full Stack Development program.",
                },
                "ref": null,
                "rendered": "Justin Gerow, based in Chicago, is a Full Stack Web Developer who graduated from the Northwestern University Coding Boot Camp Full Stack Development program.",
                "type": "p",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Justin has spent the past 18 years working primarily in the transportation, warehousing, and logistics fields. He started his career in 2000 as a package operations supervisor for a small-parcel company located in Flint, Michigan. Justin spent eight years in this role and during this time also attended the University of Michigan-Flint, where he obtained his Bachelor of Business Administration degree, with a concentration in operations management. In May 2008, he relocated to Irving, Texas, to join a large online retail organization as an area manager in a local fulfillment center. In September 2009, Justin joined a transportation company as a driver business leader based in Gary, Indiana, and in 2014 moved into his current role with the same company, as a sales operations manager based in Chicago. In 2016, he received his Certified Supply Chain Professional (CSCP) certification from APICS, the association for supply chain management.",
                },
                "ref": null,
                "rendered": "Justin has spent the past 18 years working primarily in the transportation, warehousing, and logistics fields. He started his career in 2000 as a package operations supervisor for a small-parcel company located in Flint, Michigan. Justin spent eight years in this role and during this time also attended the University of Michigan-Flint, where he obtained his Bachelor of Business Administration degree, with a concentration in operations management. In May 2008, he relocated to Irving, Texas, to join a large online retail organization as an area manager in a local fulfillment center. In September 2009, Justin joined a transportation company as a driver business leader based in Gary, Indiana, and in 2014 moved into his current role with the same company, as a sales operations manager based in Chicago. In 2016, he received his Certified Supply Chain Professional (CSCP) certification from APICS, the association for supply chain management.",
                "type": "p",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Originally from Michigan, Justin enjoys participating in outdoor activities such as running and hiking in his free time, as well as cooking and exploring Chicago with his girlfriend, Lindsey, and 10-year-old Husky, Link. He also enjoys following auto racing — specifically, rally racing and the World Endurance Championship — and playing the occasional video game on those rainy days. Coding and development is a passion of Justin’s, and he looks forward to applying his knowledge in an exciting, evolving industry. ",
                },
                "ref": null,
                "rendered": "Originally from Michigan, Justin enjoys participating in outdoor activities such as running and hiking in his free time, as well as cooking and exploring Chicago with his girlfriend, Lindsey, and 10-year-old Husky, Link. He also enjoys following auto racing — specifically, rally racing and the World Endurance Championship — and playing the occasional video game on those rainy days. Coding and development is a passion of Justin’s, and he looks forward to applying his knowledge in an exciting, evolving industry. ",
                "type": "p",
              },
            ],
            "type": "div",
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
